source/object-retention/bucket-versioning.rst

```diff
@@ -271,25 +269,22 @@ MinIO :mc:`mc` CLI, or using an S3-compatible SDK.
 
    .. tab-item:: MinIO Console
 
-      Select the :guilabel:`Buckets` section of the MinIO Console to access 
-      bucket creation and management functions. 
-      
-      Select the bucket row from the list of buckets. You can use the
-      :octicon:`search` :guilabel:`Search` bar to filter the list. 
+      Select the :guilabel:`Buckets` section of the MinIO Console to access bucket creation and management functions. You can use the :octicon:`search` :guilabel:`Search` bar to filter the list. 
       
       .. image:: /images/minio-console/console-bucket.png
          :width: 600px
          :alt: MinIO Console Bucket Management
          :align: center
-      
-      From the :guilabel:`Bucket` view, click the :guilabel:`Versioning` field 
-      to disable versioning on the bucket.
 
-      .. image:: /images/minio-console/console-bucket-overview.png
+      Each bucket row has a :guilabel:`Manage` button that when selected opens the management view for that bucket.
+
+      .. image:: /images/minio-console/console-bucket-manage.png
          :width: 600px
-         :alt: MinIO Console Bucket minio-lifecycle-management
+         :alt: MinIO Console Bucket Management
          :align: center
 
+      Select the :guilabel:`Versioning` field and follow the modal instructions to suspend versioning in the bucket.
+
       See :ref:`minio-console-admin-buckets` for more information on bucket
       management using the MinIO Console.
```

Comment:
Subject/verb agreement on line 314-315, `results` should be `result`, as result is the verb for mutations, which is plural:

```
Any mutations to an object while versioning is suspended result in overwriting that
```

Reply:
Oooh nice catch